docs/cilium.md

```diff
@@ -0,0 +1,57 @@
+# Cilium sysext
+
+This sysext ships the [Cilium CLI](https://github.com/cilium/cilium-cli).
+
+This sysext includes a service unit file to start cilium at boot.
+
+## Usage
+
+Download and merge the sysext at provisioning time using the below butane
+snippet.  Additional install flags can be passed to cilium using the
+CILIUM_INSTALL_ARGS environment variable.
+
+The snippet includes automated updates via systemd-sysupdate.
+Sysupdate will stage updates and request a reboot by creating a flag file at `/run/reboot-required`.
+You can deactivate updates by changing `enabled: true` to `enabled: false` in `systemd-sysupdate.timer`.
+
+```yaml
+variant: flatcar
+version: 1.1.0
+
+storage:
+  files:
+    - path: /opt/extensions/cilium/cilium-v0.18.2-x86-64.raw
+      contents:
+        source: https://github.com/flatcar/sysext-bakery/releases/download/latest/cilium-v0.18.2-x86-64.raw
+    - path: /etc/sysupdate.cilium.d/cilium.conf
+      contents:
+        source: https://github.com/flatcar/sysext-bakery/releases/download/latest/clium.conf
```

Comment:
```suggestion
        source: https://extensions.flatcar.org/extensions/cilium.conf
```
Same; please check out https://github.com/flatcar/sysext-bakery?tab=readme-ov-file#making-systemd-sysupdate-work-with-github-releases for more information.